standby/src/lib.rs

```diff
@@ -894,12 +874,11 @@ impl Standby {
 
     /// Process a general event that is not of any particular type or in any
     /// particular guild.
-    #[cfg_attr(feature = "tracing", tracing::instrument(level = "trace"))]
+    #[tracing::instrument(level = "trace")]
```

Comment:
Only `trace` level events are recorded inside, so there's no need for the instrument to be at the trace level.
```suggestion
    #[tracing::instrument]